src/settingsMap.ts

```diff
@@ -0,0 +1,38 @@
+class SettingsMap {
+    private settings: Map<string, Map<string, string>>
+
+    constructor() {
+        this.settings = new Map<string, Map<string, string>>()
+    }
+
+    public get(guildId: string, setting: string) : string {
+		return this.settings.get(guildId)?.get(setting) ?? ""
```

Comment:
Instead of returning an empty string, you could return null. It's more readable and logical IMO.